dbt/adapters/bigquery/connections.py

```diff
@@ -320,7 +320,7 @@ def format_rows_number(self, rows_number):
         return f"{rows_number:3.1f}{unit}".strip()
 
     @classmethod
-    def get_google_credentials(cls, profile_credentials) -> GoogleCredentials:
+    def get_google_credentials(cls, profile_credentials) -> GoogleCredentials.Credentials:
```

Comment:
This was failing `mypy`. I'm not sure how it wouldn't have failed in the past.

Reply:
I think they changed things recently because I _believe_ this is what caused the error on 1.3. Must have been an upstream spec change because our code didn't on 1.3. Good catch on the solution! Module vs. type